backend/app/api/ai_agent.py

```
@@ -0,0 +1,281 @@
+
+
+# ── 4.1 GET /activity/summary ─────────────────────────────────────────────────
+
+
+@router.get(
+    "/activity/summary",
+    response_model=ActivitySummaryOut,
+    dependencies=[Depends(get_current_user)],
+)
+async def get_activity_summary(
+    range: SummaryRange = Query(default="today"),
```

Ruff (0.15.10)

[error] Function argument `range` is shadowing a Python builtin

(A002)